main-app/app/scripts/directives/gameState.js

```diff
@@ -0,0 +1,9 @@
+(function () {
+    'use strict';
+    angular.module('Tombola.NoughtAndCrosses')
```

Comment:
Why directives? Both this and (more so) the game board represent decent chunks of functionality. There is a better construct out there.